node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,7 @@
+
+const readAny = process.argv[2];
+const fs = require('fs');
```

Comment:
It'll still work but we want to put all `require`s at the top of the file, they should be the first thing in the file.